yml/imsim.yaml

```diff
@@ -8275,3 +8275,51 @@ tables:
     datatype: long
     mysql:datatype: BIGINT
     description:
+- name: match_ref
```

Comment:
Every other relation/table in this file is lowercaseCamel.

If you're curious what this table's friends are going to look like, go to https://data-int.lsst.cloud/portal select the collection called dp02_test_preops blah blah blah and scroll through the tables